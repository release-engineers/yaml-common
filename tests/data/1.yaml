example-common:
  # common comment
  depth:
    more-depth:
      array:
      - yes
      - for sure
      str: common string
    int: 123
    float: 1.23
  bool: true
  null:
example-specific:
  depth:
    more-depth:
      # specific comment to 1.yaml
      str-specific: 1.yaml
example-exists:
  str-2: This property exists in all files, and is overridden by 1.yaml
example-mixed:
  array-common:
  - yes
  - for sure again
  array-specific:
  - no
  - because this is specific to 1.yaml
  str-common: common string
  str-specific: 1.yaml
  sre-common-multiline: |
    -----BEGIN PUBLIC KEY-----
    01234567890
    -----END PUBLIC KEY-----
  unique-to-1: this property is unique to 1.yaml
