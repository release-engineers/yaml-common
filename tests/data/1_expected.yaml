example-specific:
  depth:
    more-depth:
      # specific comment to 1.yaml
      str-specific: 1.yaml
example-exists:
  str-2: This property exists in all files, and is overridden by 1.yaml
example-mixed:
  array-specific:
  - no
  - because this is specific to 1.yaml
  str-specific: 1.yaml
  unique-to-1: this property is unique to 1.yaml
